### ISSUE TITLE
[8.0.0-svg] walkontable.selection e2e test do not pass only on Firefox

### ISSUE THREAD

**Issue by aninde** [handsontable/handsontable#6644]
### Description
One walkontable.selection test do not pass only in Firefox. It passes on Chrome, Safari and Edge. 

```
Walkontable.Selection > should apply changed settings of a highlight (mobile browser)
Expected '' to equal 'rgb(255, 0, 0)'.
Expected '' to equal 'rgb(255, 0, 0)'.
Expected '' to equal 'rgb(127, 124, 0)'.
Expected '' to equal 'rgb(127, 124, 0)'.
```
<img width="667" alt="Screenshot 2020-01-16 at 09 58 41" src="https://user-images.githubusercontent.com/30186818/72509684-569aae80-3848-11ea-99b5-b26534b7ba1a.png">

### Steps to reproduce

 run newly builded walkontable.dump after building handsontable and walkontable in Firefox

### Your environment
* Handsontable version: 8.0.0-svg builded from branch origin/feature/issue-6064)
* Browser Name and version: Firefox 72
* Operating System: macOS

**Comment by warpech** [comment]
Fixed in 9507c2dbcf66a2f8ff1e8d9c4d053d6160195b3d, as part of https://github.com/handsontable/handsontable/pull/6157

**Comment by aninde** [comment]
Thank you for the quick repair, everything works on Firefox, Safari, Chrome and Edge. I close the issue.
<img width="350" alt="Screenshot 2020-01-16 at 13 41 19" src="https://user-images.githubusercontent.com/30186818/72526139-03375900-3866-11ea-9666-155ae5d21ccf.png">
